easybuild/easyconfigs/l/libfabric/libfabric-1.15.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'ConfigureMake'
+
+name = 'libfabric'
+version = '1.15.0'
```

Comment:
we should probably wait for 1.15.1 since the psm3 issue should be fixed there:
https://github.com/ofiwg/libfabric/issues/7735#issuecomment-1118908953